src/blocks/AnimatedPhoneBlock.tsx

```diff
@@ -0,0 +1,153 @@
+    textAlign: textPosition === 'center' ? 'center' : 'left',
+    width: '100%',
+    paddingRight: textPosition === 'left' ? '7rem' : '0',
+    paddingLeft: textPosition === 'right' ? '7rem' : '0',
+    [TABLET_BP_DOWN]: {
+      paddingRight: 0,
+      paddingLeft: 0,
+    },
+  }),
+)
+
+const Title = styled('h2')(
+  ({
+    size,
+    displayOrder,
+  }: SectionSizeProps & { displayOrder: 'top' | 'bottom' }) => ({
+    fontSize: size === 'xl' ? '4.5rem' : '2.5rem',
+    wordWrap: 'break-word',
+    width: '100%',
+    [TABLET_BP_DOWN]: {
+      fontSize: size === 'xl' ? '3.75rem' : '2rem',
+      marginTop: displayOrder === 'top' ? '3rem' : '1.414rem',
+    },
+  }),
+)
+
+const Paragraph = styled('div')(({ size }: SectionSizeProps) => ({
+  fontSize: size === 'xl' ? '1.125rem' : '1rem',
+  wordWrap: 'break-word',
+  marginTop: '1.5rem',
+}))
+
+const PhoneContainer = styled('div')(
+  ({
+    alignment,
+    displayOrder,
+  }: {
+    alignment: string
+    displayOrder: 'top' | 'bottom'
+  }) => ({
+    padding: '0 30px',
+    width: '100%',
+    '&:>div': {
+      width: '100%',
+    },
+    display: alignment === 'center' ? 'none' : 'block',
+    [TABLET_BP_DOWN]: {
+      maxWidth: '100%',
+      width: 'auto',
+      marginTop: displayOrder === 'top' ? '0' : '3rem',
+      display: 'block',
+      order: displayOrder === 'top' ? -1 : 'initial',
+    },
+  }),
+)
+
+interface AnimatedPhoneBlockProps extends BaseBlockProps {
+  title: string
+  paragraph: MarkdownHtmlComponent
+  text_position: TextPosition
+  button_title: string
+  button_type: 'filled' | 'outlined'
+  button_link: LinkComponent
+  show_button: boolean
+  size: SectionSize
+  media_position: 'top' | 'bottom'
+}
+
+export const AnimatedPhoneBlock: React.FunctionComponent<
+  AnimatedPhoneBlockProps
+> = ({
+  title,
+  paragraph,
+  text_position,
+  button_title,
+  button_type,
+  button_link,
+  show_button,
+  color,
+  size,
+  media_position,
+}) => {
+  return (
+    <SectionWrapper color={color && color.color} size={size}>
+      <AlignableContentWrapper textPosition={text_position}>
+        <TextWrapper textPosition={text_position}>
+          <Title size={size} displayOrder={media_position}>
+            {title}
+          </Title>
+          <Paragraph
+            size={size}
+            dangerouslySetInnerHTML={{
+              __html: paragraph && paragraph.html,
+            }}
+          />
+          {show_button && (
+            <ButtonLinkWithMargin
+              href={getStoryblokLinkUrl(button_link)}
+              type={button_type}
+              size="sm"
+              bold
+            >
+              {button_title}
+            </ButtonLinkWithMargin>
+          )}
+        </TextWrapper>
+        <PhoneContainer alignment={text_position} displayOrder={media_position}>
+          <LazyLottie
+            options={{
+              animationData: import('animations/chatDemo.json'),
```

Comment:
and same here, could we add a chunk name? https://github.com/webpack/webpack/tree/master/examples/code-splitting-specify-chunk-name